modules/rabbitmq/manager.go

```diff
@@ -0,0 +1,222 @@
+package rabbitmq
+
+import (
+	"errors"
+	"sync"
+
+	"github.com/CodeCollaborate/Server/utils"
+	"github.com/streadway/amqp"
+)
+
+/**
+ * RabbitMq manager for CodeCollaborate Server.
+ * @author: Austin Fahsl and Benedict Wong
```

Comment:
I felt that we actually probably don't want author tags, since that would imply ownership. If there's any left, we probably should remove those.